modules/apt.sh

```diff
@@ -5,7 +5,8 @@ private_repo_url() {
     local credentials="$2"
     local file="$3"
 
-    echo "https://${credentials}@${repo_url}/ubuntu${file}"
+    local repo_schema="${repo_url/:\/\/*/}}"
```

Comment:
I think you meant `repo_scheme`, or `url_scheme`: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Syntax

Reply:
yes, good catch, fixed